src/Network/Probe.php

```diff
@@ -1411,6 +1411,37 @@ private static function pumpio($webfinger, $addr)
 		return $data;
 	}
 
+	/**
+	 * @brief Check for twitter contact
```

Comment:
Shouldn't this be part of the addon that deals with Twitter contacts and not the core of Friendica?

Reply:
The problem is that someone shares a Twitter post. This post is then shared to other systems. These systems possibly don't have activated Twitter at all. So we need some basic support in the core.